packages/node_modules/samples/browser-meetings/app.js

```diff
@@ -168,12 +169,15 @@ function joinMeeting(meetingId) {
   if (!meeting) {
     throw new Error(`meeting ${meetingId} is invalid or no longer exists`);
   }
+  const resourceId = webex.devicemanager._pairedDevice ?
+    webex.devicemanager._pairedDevice.identity.id :
+    undefined;
 
   meeting.join({
     pin: meetingsJoinPinElm.value,
     moderator: meetingsJoinModeratorElm.checked,
     moveToResource: false,
-    resourceId: webex.devicemanager._pairedDevice.identity.id
+    resourceId
   })
```

Comment:
Thanks for implementing this hotfix based on my previously merged PR! good work!